frontend/packages/client/src/components/CommunityProposals.js

```diff
@@ -86,7 +86,7 @@ export default function CommunityProposals({ community = { id: 1 } }) {
         {notMobile && (
           <div className="columns m-0">
             <div className="column p-0 is-10">
-              <Link to="/proposal/create">
+              <Link to={`/proposal/create?communityId=${communityId}`}>
```

Comment:
when proposal creation starts we need a way to pass the communityId from where the flow was triggered